build/batchcopy.cmd

```diff
@@ -1 +1,14 @@
-@powershell -NoProfile -ExecutionPolicy unrestricted -File "%~dp0\batchcopy.ps1" %*
+@echo off
+
+set SRC=%1
+set DST=%2
+
+set SRC=%SRC:/=\%
+set DST=%DST:/=\%
+
+if not exist %SRC% goto end
```

Comment:
Not sure, just ported it from PowerShell. I don't see a reason why we couldn't create the directory here, but in general it's not expected that this command would ever be run without having done a build previously.